src/fragment-components/tiles/a-expandable-tile.tsx

```diff
@@ -0,0 +1,205 @@
+	reactClassNamesFromComponentObj
+} from '../../utils/fragment-tools';
+
+export const AExpandableStyleUI = ({ selectedComponent, setComponent }: any) => {
+	return <>
+		<TileMorphism component={selectedComponent} setComponent={setComponent} />
+		<Checkbox
+			labelText='Light theme'
+			id='theme-select'
+			checked={selectedComponent.light}
+			onChange={(checked: any) => {
+				setComponent({
+					...selectedComponent,
+					light: checked
+				});
+			}}
+		/>
+		<Checkbox
+			labelText='Expanded'
+			id='expanded'
+			checked={selectedComponent.expanded}
+			onChange={(checked: any) => {
+				setComponent({
+					...selectedComponent,
+					expanded: checked
+				});
+			}}
+		/>
+		<Checkbox
+			labelText='Show outline'
+			id='outline'
+			checked={selectedComponent.outline}
+			onChange={(checked: any) => {
+				setComponent({
+					...selectedComponent,
+					outline: checked
+				});
+			}}
+		/>
+		<ComponentCssClassSelector componentObj={selectedComponent} setComponent={setComponent} />
+	</>
+};
+
+export const AExpandableTileCodeUI = ({ selectedComponent, setComponent }: any) => {
+	return <>
+		<TextInput
+			value={selectedComponent.codeContext?.name}
+			labelText='Input name'
+			onChange={(event: any) => {
+				setComponent({
+					...selectedComponent,
+					codeContext: {
+						...selectedComponent.codeContext,
+						name: event.currentTarget.value
+					}
+				});
+			}}
+		/>
+	</>
+};
+
+const showOutlineStyle = css`
+	span.bx--tile-content__above-the-fold,
+	span.bx--tile-content__below-the-fold {
+		min-height: 16px;
+		outline: 1px dashed #78a9ff;
+	}
+`;
+
+export const AExpandableTile = ({
+	children,
+	componentObj,
+	onDrop,
+	selected,
+	...rest
+}: any) => {
+
+	// Splicing bottomFold from children so children can be appended to above the fold content
+	const foldIndex = children.findIndex(({ props }: any) => (props !== undefined ? (props.componentObj.type === 'tilefold') : false));
+	const bottomFold = children.splice(foldIndex, 1);
+
+	return <AComponent
+		componentObj={componentObj}
+		selected={selected}
+		headingCss={css`display: block;`}
+		{...rest}>
+		<ExpandableTile
+			light={componentObj.light}
+			className={`${componentObj.cssClasses?.map((cc: any) => cc.id).join(' ')} ${componentObj.outline ? showOutlineStyle : ''}`}
+			expanded={componentObj.expanded}>
+			<TileAboveTheFoldContent onDrop={onDrop}>{children}</TileAboveTheFoldContent>
+			{
+				// Render bottom fold component
+				bottomFold
+			}
+		</ExpandableTile>
+	</AComponent>;
+};
+
+// Splits data into folds - all exports will have a common approach
+const getFoldObjects = (json: any, jsonToTemplate: any) => {
+	// Destructuring existing items to prevent changing default object
+	const items = [...json.items];
+
+	// Find tileFold index & seperate from list from the list
+	const tileFoldIndex = items.findIndex((item: any) => item.type === 'tilefold');
+	const tileFoldItems = items.splice(tileFoldIndex, 1);
```

Comment:
Probably missing something, but if this is getting a single item out of the items array, why not just
```
const tileFoldIndex = json.items.findIndex((item: any) => item.type === 'tilefold');
tileFoldItems = json.items[tileFoldIndex];
```
without needing to make the copy of json.items?